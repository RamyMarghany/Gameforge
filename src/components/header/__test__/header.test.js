import React from 'react';
import ReactDOM from 'react-dom'
import Header from './../index'
import "@testing-library/jest-dom"; 
import renderer from 'react-test-renderer'

it('render without crashing', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Header></Header>, div)
})


it('matches snapshots', ()=>{
    const tree = renderer.create(<Header/>).toJSON()
    expect(tree).toMatchSnapshot();
})