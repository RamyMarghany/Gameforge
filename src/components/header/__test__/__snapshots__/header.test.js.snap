// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<div
  className="stylecss__HeaderContainer-sc-17bqppt-0 hUAtGD"
  data-testid="header"
>
  <div
    className="stylecss__HeaderWrapper-sc-17bqppt-1 gFUKaM"
  >
    <a
      className="stylecss__Logo-sc-17bqppt-2 edXTst"
    >
      Gameforge
    </a>
    <div
      className="stylecss__DropdownContainer-sc-5d7cru-0 enAiie Dropdown"
      data-testid="dropdown"
    >
      <button
        className="stylecss__DropdownButton-sc-5d7cru-1 uwfyr"
        onClick={[Function]}
      >
        Settings
      </button>
      <div
        className="stylecss__DropdownList-sc-5d7cru-2 fCNmcA dropdown-group"
      >
        <a
          className="stylecss__DropdownItem-sc-5d7cru-3 eSRCjt"
        >
          Profile
        </a>
        <a
          className="stylecss__DropdownItem-sc-5d7cru-3 eSRCjt"
        >
          settings
        </a>
        <a
          className="stylecss__DropdownItem-sc-5d7cru-3 eSRCjt"
        >
          Log out
        </a>
      </div>
    </div>
  </div>
</div>
`;
